cli/cmd/kubectl-bind/cmd/kubectlBind.go

```diff
@@ -70,5 +70,5 @@ func KubectlBindCommand() *cobra.Command {
 	}
 	bindCmd.AddCommand(apiserviceCmd)
 
-	return root
+	return bindCmd
```

Comment:
wrong commit?

Reply:
not quite. I was missing one more commit to add `bindCmd` to `rootCmd`. added now